app/src/ui/app-error.tsx

```diff
@@ -53,13 +49,13 @@ export class AppError extends React.Component<IAppErrorProps, IAppErrorState> {
   public constructor(props: IAppErrorProps) {
     super(props)
     this.state = {
-      error: props.errors[0] || null,
+      error: props.error,
       disabled: false,
     }
   }
 
   public componentWillReceiveProps(nextProps: IAppErrorProps) {
```

Comment:
You may very well be right. I was trying to be the least impactful to this component... and didn't know exactly the bug this state management was supposed to prevent - maybe some scenario I don't know to test. I assume it happened when a user had back to back errors.. which can still happen here.  I am kind of keen on leaving this alone unless i have time to really investigate what that change might mean as I don't think it is needed for this work.